aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageDownloadFileOperation.java

```diff
@@ -97,7 +100,7 @@ public void onStateChanged(int transferId, TransferState state) {
                     @SuppressWarnings("checkstyle:MagicNumber")
                     @Override
                     public void onProgressChanged(int transferId, long bytesCurrent, long bytesTotal) {
-                        int percentage = (int) (bytesCurrent / bytesTotal * 100);
+                        // int percentage = (int) (bytesCurrent / bytesTotal * 100);
```

Comment:
Should we just get rid of this?

Reply:
I'll just leave these like they were, for this PR, and remove my added `//`.